shipyard/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Shipyard logs to Datadog dashboard",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog connection settings in your Shipyard dashboard",
+        "image_url": "images/datadog-input.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Shipyard logo in teal",
+        "image_url": "images/shipyard-teal.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Automation",
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Cloud",
+      "Category::Collaboration",
+      "Category::Google Cloud",
+      "Category::Issue Tracking",
+      "Category::Orchestration",
+      "Category::Containers",
+      "Category::Developer Tools",
+      "Category::Kubernetes",
+      "Category::Testing",
+      "Submitted Data Type::Logs",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "dashboards": {},
```

Comment:
Once you move the dashboard JSON content to the right file, you should link that file in this manifest.json file. Example: https://github.com/DataDog/integrations-extras/blob/master/akamai_datastream_2/manifest.json#L46